client/src/main/java/io/pravega/client/ClientConfig.java

```diff
@@ -108,6 +108,15 @@
      */
     private final MetricListener metricListener;
 
+    /**
+     * An optional property representing whether the current run is for testing purpose. By default this property is
+     * not enabled.
+     *
+     * If this property is set, the request timeout to segmentstore is set to a very large value(3600s) in order to not
+     * mask any bugs. By default the request timeout in real application is set to 30s.
+     */
+    private final boolean enableTesting;
```

Comment:
This property is too vague.
Now it means that we will use a longer timeout, but the user cannot know what this property will mean in the future.

It is better to name it 'request timeout'  or something like that.